src/enum/modals/createModal.ts

```diff
@@ -1,4 +1,4 @@
-export enum CreateModalEnum {
+export enum createModalEnum {
```

Comment:
Is there a reason why you changed the case of the enums? I'd suggest keeping the CamelCase

Reply:
Sure, I was just confused which one should we use